app/views/landing_page/blocks/_hero.html.erb

```diff
@@ -1,8 +1,14 @@
 <%
   add_view_stylesheet("landing_page/hero")
+
+  hero_classes = "app-b-hero"
+  hero_classes << " app-b-hero--mid-page" if block.data["mid_page"]
```

Comment:
Rails suggestion to avoid needing a leading space on the extra classes:

```Ruby
<%
  hero_classes = %w[app-b-hero]
  hero_classes << "app-b-hero--mid-page" if block.data["mid_page"]

  hero_content_classes = %w[app-b-hero__textbox]
  hero_content_classes << "app-b-hero__textbox--mid-page" if block.data["mid_page"]
%>
# ...

<%= content_tag("div", class: hero_classes) do %>
  # ...
  <%= content_tag("div", class: hero_content_classes) do %>
    # ...
  <% end %>
<% end %>
```

Reply:
Great thanks, that's much better